src/Basque/Mews.Fiscalizations.Basque/Model/Invoice/InvoiceHeader.cs

```diff
@@ -11,18 +11,16 @@ public sealed class InvoiceHeader
         public InvoiceHeader(
             String1To20 number,
             DateTime issueDate,
-            DateTime issueDateTime,
-            bool isSimplified = false,
-            bool issuedInSubstitutionOfSimplifiedInvoice = false,
+            bool? isSimplified = null,
+            bool? issuedInSubstitutionOfSimplifiedInvoice = null,
             String1To20 series = null,
             CorrectingInvoice correctingInvoice = null,
             IEnumerable<CorrectedInvoice> correctedInvoices = null)
         {
             Number = number;
             IssueDate = issueDate;
-            IssueDateTime = issueDateTime;
-            IsSimplified = isSimplified;
-            IssuedInSubstitutionOfSimplifiedInvoice = issuedInSubstitutionOfSimplifiedInvoice;
+            IsSimplified = isSimplified.ToOption();
```

Comment:
Should this be an option really though? It sounds like a Yes or no type of thing.